content/issues/1/weaving-interface/style.css

```diff
@@ -0,0 +1,198 @@
+:root {
+    /* css variables duplicating site scss variables */
+    --red: #CE2949;
+    --off-white: #F6F6F6;
+    --purple: #4C2A85;
+    --light-purple: #E0CEFF;
+
+}
+
+body.article .deepzoom {
+    width: 100vw;
+    height: 230px;
+    margin-top: 15vh;
+    margin-bottom: 30vh;
+    background-color: black;
+
+    counter-reset: highlightZoneCounter;
+}
+
+/* breakpoint-s */
```

Comment:
could always put these breakpoint values in the CSS vars block up top too

Reply:
I tried that and for some reason the variables didn't work in the media queries! I didn't mess with it for very long or investigate, though.

Reply:
i kinda figured you had tried and it was just some syntax thing. let's not worry about it for now